commands/operator-sdk/cmd/root.go

```diff
@@ -23,7 +23,7 @@ import (
 func NewRootCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     "operator-sdk",
-		Short:   "A sdk for building operator with ease",
+		Short:   "A SDK for building operators with ease",
```

Comment:
Minor nit but while we're cleaning up grammar this should be: `An SDK ...`

Reply:
@jplock True, but we're pronouncing the acronym SDK here, not the words "software development kit".  
When you pronounce "S" it begins with a vowel sound "ess", hence "An SDK".

I won't split hairs over this though since grammar is weird and I might be wrong :)

Reply:
I actually looked this up because I've always been curious what the correct way is. @hasbro17 is correct. It's based on how it sounds. So SDK is, "Ess Dee Kay", hence using "An", but LAN is pronounced "Lan" as one word, so it would be preceded by "A".

Link if you're curious:
http://blog.apastyle.org/apastyle/2012/04/using-a-or-an-with-acronyms-and-abbreviations.html